postcode_lookup.R

```diff
@@ -0,0 +1,149 @@
+#####################################################
+# Postcode Lookup
+# Author: Catherine Holland
+# Date: February 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input -
+# Description -
+#####################################################
+
+library(dplyr)
+library(tidyr)
+library(haven)
+
+
+## read data in ##
+
+# postcode data
+pc_file <- read_spd_file()
+
+# simd data
+read_simd_file <- function(file) {
+  simd_path <- fs::path(get_lookups_dir(), "Deprivation", file)
+  # If given a sav extension (or other), swap it for rds
+  simd_path <- fs::path_ext_set(simd_path, "rds")
+  # Check if the file exists and we can read it
+  if (!fs::file_access(simd_path, "read")) {
+    rlang::abort(message = "Couldn't read the simd file")
+  }
+  return(simd_path)
+}
+simd_file <- readr::read_rds(read_simd_file(file = "postcode_2021_2_simd2020v2.rds"))
+
+# locality
+read_locality_file <- function(file) {
+  locality_path <-
+    fs::path(get_lookups_dir(), "Geography", "HSCP Locality", file)
+  # If given a sav extension (or other), swap it for rds
+  locality_path <- fs::path_ext_set(locality_path, "rds")
+  # Check if the file exists and we can read it
+  if (!fs::file_access(locality_path, "read")) {
+    rlang::abort(message = "Couldn't read the locality file")
+  }
+  return(locality_path)
+}
```

Comment:
To its own script please, as above for naming, maybe: `get_locality_lookup_path`?

Reply:
These functions exist but needed a change for it to work - @Jennit07 was on this edit - the functions were included until this was updated. Would you want this name changed on that function?